vertica_python/vertica/cursor.py

```diff
@@ -117,7 +117,7 @@
     file_type = file_type + (file,)
 
 
-RE_NAME_BASE = u"[a-zA-Z_][\\w\\d\\$_]*"
+RE_NAME_BASE = u"[0-9_a-zA-Z_][\\w\\d\\$_]*"
```

Comment:
Is '_' after '0-9' really needed?

Reply:
@sitingren Yeah that was just a typo, good catch.The latest commit does address this.